WISE-Community/WISE-Client
# chore(Teacher): Convert Main Menu

## Changes

- Converted the Main Menu that pops out from the left side of the Classroom Monitor and Authoring Tool. This also fixes the problem of the Main Menu not working due to a recent change made to the Side Menu.
- Changed Side Menu to call state.go() itself instead of emitting an event to its parent.

## Test

- Make sure the Main Menu works in the Classroom Monitor and Authoring Tool
- Make sure the Side Menu works in the Classroom Monitor and Authoring Tool

Closes #813
